### PR TITLE
Feature/corelogic

### PR DESCRIPTION
**Pull request by drawyan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
 # to-start-case
 Convert a string to Start Case for human display purposes.
+
+Converts a string, including strings in camelCase or snake_case, into Start Case (a variant
+of Title Case where all words start with a capital letter), it keeps original single quote
```
Comment (onlywei) [inline README.md]: The last part of this documentation (after the comma) should be a separate sentence and very strongly note that "UNLIKE OTHER Start Case libs: ... "

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+```js
+import toStartCase from 'to-start-case';
+
+toStartCase('Mangement_companies'); // 'Management Companies'
+toStartCase(`hell's kitchen`); // `Hell's Kitchen`
+```
+
+Note that this is written entirely in ES2015+. If you need to use this in a runtime that does not
+support ES2015 features, you will need to provide your own transpilation process in order to use
+this library.
```
Comment (onlywei) [inline README.md]: This note at the bottom is actually not true anymore

File: package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  "name": "to-start-case",
+  "version": "1.0.0",
+  "description": "Convert a string to Start Case for human display purposes.",
+  "main": "dist/to-start-case.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "build": "node tools/build.js",
+    "prepare": "npm run build",
```
Comment (onlywei) [inline package.json]: We should still use "prepublish". "prepare" was a mistake.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "keywords": [
+    "convert",
+    "conversion",
+    "string",
+    "start",
+    "case"
+  ],
+  "contributors": [
+    "Zhuoyuan Zhang",
```
Comment (onlywei) [inline package.json]: I think there's a way to include email addresses in this list

File: package.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "to-start-case",
+  "version": "1.0.0",
+  "description": "Convert a string to Start Case for human display purposes.",
+  "main": "dist/to-start-case.js",
```
Comment (onlywei) [inline package.json:5]: This "main" field is not enough. I need you to understand what the other fields "module", "browser", "esnext" do and what they are for.

**Review by onlywei — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (drawyan) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (drawyan) [reply README.md]: Removed

File: package.json (hunk 5 above)
Comment (drawyan) [reply package.json:5]: Done

File: package.json (hunk 3 above)
Comment (drawyan) [reply package.json]: Done

File: package.json (hunk 4 above)
Comment (drawyan) [reply package.json]: Done

**Pull request merged**